src/components/pages/Volunteers/index.tsx

```diff
@@ -79,20 +107,69 @@ const UnconnectedVolunteers: React.FC<PropsFromRedux> = ({
   };
 
   useEffect(() => {
-    if (!hasFetchedVolunteers) {
-      loadVolunteers();
+    if (
+      !hasFetchedVolunteers &&
+      volunteers.all_volunteers.length === 0 &&
```

Comment:
same here ! probaaably remove the all volunteers length. if there are 0, think this creates an infinite looop